includes/Models/DataStore/VendorOrderStatsStore.php

```
@@ -0,0 +1,161 @@
+     */
+    public function get_sales_chart_data( string $start_date, string $end_date ): array {
+        global $wpdb;
+
+        $this->clear_all_clauses();
+        $this->add_sql_clause( 'select', 'SUM(wos.total_sales) as total_sales,' );
+        $this->add_sql_clause( 'select', 'SUM(wos.net_total) as net_sales,' );
+        $this->add_sql_clause( 'select', 'SUM(dos.admin_commission) as commissions,' );
+        $this->add_sql_clause( 'select', 'COUNT(dos.order_id) as order_count' );
+        $this->add_sql_clause( 'from', $this->get_table_name_with_prefix() . ' dos' );
+        $this->add_sql_clause( 'join', "INNER JOIN {$wpdb->prefix}wc_order_stats wos ON dos.order_id = wos.order_id" );
+        $this->add_sql_clause( 'where', " AND wos.status NOT IN ( '" . implode( "','", $exclude_order_statuses ) . "' )" );
```

PHPMD (2.15.0)

Avoid unused local variables such as '$exclude_order_statuses'. (Unused Code Rules)

(UnusedLocalVariable)